src/iconid.rs

```diff
@@ -143,7 +143,9 @@ fn apply_location_based_substitution(
 
         for sub in feature_table_substitution.substitutions() {
             let alt = sub.alternate_feature(feature_table_substitution.offset_data())?;
-            for lookup_idx in alt.lookup_list_indices() {
+            let mut lookup_list_indices: Vec<BigEndian<u16>> = alt.lookup_list_indices().to_vec();
```

Comment:
I would map to u16 not capture `Vec<BigEndian>`

I think this merits some sort of citation

Reply:
done